packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -430,6 +430,9 @@ abstract class ResidentRunner {
     this.ipv6,
   }) {
     _mainPath = findMainDartFile(target);
+    if (target.contains(other)) {
+      
+    }
```

Comment:
I presume this was a mistake?

Reply:
this is a mistake, but I'm going to close this PR because the material spec has changed to "black" for default icon colors, thanks for your review